src/mdx-components.tsx

```diff
@@ -11,6 +11,7 @@ export function useMDXComponents(components: MDXComponents): MDXComponents {
     h3: ({children}) => (
       <h3 className="leading-tight tracking-tight text-gray-900">{children}</h3>
     ),
+    p: ({children}) => <p className="text-gray-900 w-96">{children}</p>,
```

Comment:
In globals.css, the <p> already got a width of  `max-w-3xl`. I think I like this place better because in global.css it could have side effects like the styling of unordered lists. Can you pick one place for the width?

Reply:
Another thought: Do you only want to set the width of the `<p>` or the whole markdown content? What if there are long headers? If you want a max width for the whole markdown block you could also set it in `src/components/localized-markdown.tsx`

Reply:
I prefer the <p> in the mdx-components.tsx.